recipes/libx11/all/conanfile.py

```diff
@@ -0,0 +1,62 @@
+import os
+import subprocess
+import shlex
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+def remove_prefix(s, prefix):
+    return s[len(prefix):] if s.startswith(prefix) else s
+
+
+class LibX11Conan(ConanFile):
+    name = "libx11"
+    license = "X11"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.x.org/wiki/"
+    description = "Client interface to the X Window System, otherwise known as \'Xlib\'",
+    settings = "os", "compiler", "build_type", "arch"
+    _required_system_package = "libx11-dev"
+    _system_package_tool = None
+
+    def _system_packages(self):
+        if not self._system_package_tool:
+            self._system_package_tool = tools.SystemPackageTool()
+        return self._system_package_tool
+
+    def system_requirements(self):
+        installer = self._system_packages()
+        if not installer.installed(self._required_system_package):
+            raise ConanInvalidConfiguration(
```

Comment:
Is the plan to have `libx11-dev` installed in C3I build configurations ? Otherwise that would drastically reduce this package's usefulness, because it would not be possible to require it from any CCI recipe.

Reply:
Yes, they should be installed in CCI, if not all these kinds of recipes would be useless as you say.